e2e/cypress/common/namespace.ts

```diff
@@ -1,6 +1,33 @@
+import { HOST } from './constants';
+
 export function selectNamespace(namespace: string) {
   cy.gcy('search-select').click();
   cy.gcy('search-select-new').click();
   cy.gcy('namespaces-select-text-field').type(namespace);
   return cy.gcy('namespaces-select-confirm').click();
 }
+
+export function enableNamespaces(projectId: number, enable = true) {
+  cy.url().then((originalPage) => {
+    cy.visit(`${HOST}/projects/${projectId}/manage/edit`);
+    const checkbox = cy
+      .gcy('project-settings-use-namespaces-checkbox')
+      .find('input[type=checkbox]');
+
+    if (enable) {
+      checkbox.check();
+    } else {
+      checkbox.uncheck();
+    }
+
+    cy.contains('button', 'Save').click();
+
+    if (originalPage !== 'about:blank') {
+      cy.visit(originalPage);
+    }
+  });
```

Comment:
This should be an API call or ideally baked into project test data. This way it will slow down a lot of tests.